src/main/io/ledstrip.c

```diff
@@ -484,20 +478,32 @@ static void applyLedFixedLayers()
         }
 
         if (ledGetOverlayBit(ledConfig, LED_OVERLAY_THROTTLE)) {
-            hOffset += scaledAux;
+			if (rcData[THROTTLE] < (PWM_RANGE_MIN + PWM_RANGE_MAX)/2) //if below average
```

Comment:
The code shouldn't be hardcoded to use `THROTTLE`, but use `scaledAux`, so that it can be assigned to any of the sticks / aux channels: